From 8dde58e01621d2996ae1497144b7b0202fee021b Mon Sep 17 00:00:00 2001
From: Katsuya Matsubara <matsu@igel.co.jp>
Date: Fri, 20 Jul 2012 17:42:49 +0900
Subject: [PATCH 42/97] gst/h264parse: prioritize legacyh264parse over
 h264parse

This changes rank of the legasyh264parse plugin into PRIMARY + 2.
Incidentally, rank of h264parse under gst/videoparsers is PRIMARY + 1.
---
 gst/h264parse/gsth264parse.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/h264parse/gsth264parse.c b/gst/h264parse/gsth264parse.c
index eeacb71..bc58269 100644
--- a/gst/h264parse/gsth264parse.c
+++ b/gst/h264parse/gsth264parse.c
@@ -2758,7 +2758,7 @@ static gboolean
 plugin_init (GstPlugin * plugin)
 {
   return gst_element_register (plugin, "legacyh264parse",
-      GST_RANK_NONE, GST_TYPE_H264PARSE);
+      GST_RANK_PRIMARY + 2, GST_TYPE_H264PARSE);
 }
 
 GST_PLUGIN_DEFINE (GST_VERSION_MAJOR,
-- 
1.7.10.4

